### PR TITLE
Removing "zenhub" and "hiberly" - websites are down

### PR DESCRIPTION
**Pull request by spekulatius**
Hey @ripienaar 

just removing two dead projects here.

Cheers,
Peter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -825,7 +824,6 @@ Table of Contents
    * [ubertesters.com](https://ubertesters.com/) — Test platform, integration and crowdtesters, 2 projects, 5 members
    * [vivifyscrum.com](https://www.vivifyscrum.com/) — Free tool for Agile project management. Scrum Compatible
    * [YouTrack](https://www.jetbrains.com/youtrack/buy/#edition=incloud) — Free hosted YouTrack (InCloud) for FOSS projects, private projects (free for 3 users). Includes time tracking and agile boards
-   * [zenhub.io](https://www.zenhub.io/) — The only project management solution inside GitHub. Free for public repos, OSS and nonprofit organizations
```
Comment (ripienaar) [inline README.md:828]: still legit, their url is now https://www.zenhub.com/ is all

Comment (spekulatius) [reply README.md:828]: Ah, okay. I didn't know. I've updated the entry @ripienaar

**Comment by ripienaar** [comment]
nice thanks

**Comment by spekulatius** [comment]
Happy to help ! :)

**Pull request merged**